netket/graph/grid.py

```diff
@@ -95,7 +113,36 @@ def __init__(self, length: List, *, pbc: bool = True, color_edges: bool = False)
         super().__init__(graph)
 
     def __repr__(self):
-        return "Grid(length={}, pbc={})".format(self.length, self.pbc)
+        if all(self.pbc):
+            pbc = True
+        elif not any(self.pbc):
+            pbc = False
+        else:
+            pbc = self.pbc
+        return "Grid(length={}, pbc={})".format(self.length, pbc)
```

Comment:
Not important, but I tend to find this more readable?

```suggestion
        return f"Grid(length={self.length}, pbc={v})"
```

Reply:
Yes. Might be a left-over from the time we supported Python versions before 3.6? (Or maybe before I realized we no longer need to support those versions.)